src/main/migrations/changesets/initial_schema.xml

```diff
@@ -0,0 +1,581 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
is this easy to regenerate? I know there have been some schema changes since this branch and it would be easier to regen than to figure out what those changes were.